vcx/libvcx/src/api/proof.rs

```diff
@@ -35,6 +72,7 @@ pub extern fn vcx_proof_create(command_handle: u32,
                                source_id: *const c_char,
                                requested_attrs: *const c_char,
                                requested_predicates: *const c_char,
+                               revocation_interval: *const c_char,
```

Comment:
Because libvcx has not yet reached 1.0 its API is still considered unstable.  Libvcx can follow whatever process the indy-sdk has for managing these kinds of changes.

Reply:
For now I suggest to allow breaking-changes here